pkg/email/templates/model_plan_suggested_phase_body.html

```diff
@@ -33,7 +33,7 @@
 <p>If you’re not ready to update your model’s status, please <a href="{{.ClientAddress}}/models/{{.ModelPlanID}}/collaboration-area/model-timeline" style="font-weight: normal">adjust your model’s anticipated timeline</a>. If you adjust your timeline, you may receive this notification again.</p>
 <hr />
 <br />
-<p>You’re receiving this notification because you are listed as the model lead for {{.ModelPlanName}}. If this is incorrect, <a href="{{.ClientAddress}}/models/{{.ModelPlanID}}/collaboration-area/collaborators" style="font-weight: normal">please update the model team</a>.</p>
+<p>You are currently subscribed to notifications about suggested model status changes. If you no longer wish to receive these email notifications, please <a href="{{.ClientAddress}}/notifications/settings?unsubscribe_email=INCORRECT_MODEL_STATUS_ACTIVITY" style="font-weight: normal">unsubscribe</a> or change your <a href="{{.ClientAddress}}/notifications/settings" style="font-weight: normal">notification settings</a>. Regardless of your settings, you will continue to receive these notifications for models where you are the Model Lead.</p>

```

Comment:
QQ, should this be using the new shared footer that Elle made? Or is the content too different?

Reply:
I believe it should be using this specific footer since it was called out in the figma, and eye test I think it is a bit too different yeah